datalist-polyfill.js

```diff
@@ -260,17 +260,21 @@
           }
 
           // the select should get positioned underneath the input field ...
-          if (inputStyles.getPropertyValue('direction') === 'rtl') {
-            dataListSelect.style.marginRight = '-' + (rects[0].width + inputStyleMarginLeft) + 'px';
+          if ( inputStyles.getPropertyValue( 'display' ) === 'block' ) {
```

Comment:
Oh, shoot, when I merged in changes, I didn't style the code from `master` with the new style. I'll take care of that after merging this in and adding you as a contributor :)